src/scene/camera.js

```diff
@@ -465,6 +495,11 @@ class Camera {
         return this._projMatSkybox;
     }
 
+    getExposure() {
+        const ev100 = Math.log2((this._aperture * this._aperture) / this._shutter * 100.0 / this._sensitivity);
+        return 1.0 / (Math.pow(2.0, ev100) * 1.2);
```

Comment:
I can find references for calculating the exposure value (`ev100`) and I wonder what `1.2` means here. Is it like a magic number to make things look right?

Reply:
It's from the filament documentation: https://google.github.io/filament/Filament.md.html#imagingpipeline/physicallybasedcamera